scripts/main_obs_prep.sh

```diff
@@ -1,31 +1,59 @@
 #!/bin/bash -l
+
 echo 'main_obs_prep.sh starts'
 echo ${ROOT_GODAS_DIR}
 echo CDATE is $CDATE
 
 ObsRunDir=$RUNCDATE/Data/${CDATE}    #Path for observations to be ingested by DA
-mkdir -p ${ObsRunDir}
 
 # Prep absolute dynamic topography obs
 source ${ROOT_GODAS_DIR}/scripts/adt_prep_obs.sh 
+echo "ADT done"
 
 # Prep T&S profile obs
 source ${ROOT_GODAS_DIR}/scripts/fnmoc_prep_obs.sh
+echo "FNMOC done"
 
 # Prep ice concentration obs
-source ${ROOT_GODAS_DIR}/scripts/icec_prep_obs.sh
+#source ${ROOT_GODAS_DIR}/scripts/icec_prep_obs.sh
 
 # Prep sst obs
 ListOfSST="sst.windsat_l3u.ghrsst \
            sst.gmi_l3u.ghrsst \
            sst.amsr2_l3u.ghrsst
-           sst.avhrrmta_l3u.nesdis \
-           sst.avhrr19_l3u.nesdis \
            sst.viirs_l3u.nesdis "
```

Comment:
Why was it removed?

Reply:
The directory name for AVHRR data is sst.avhrr_l3u.nesdis. This directory has the data files for both the sensors - AVHRR19 and AVHRRMTA.